src/Honeybee_createHBZones.py

```diff
@@ -89,8 +89,17 @@ def main(zoneName,  HBZoneProgram, HBSurfaces, isConditioned):
     # call the surface from the hive
     hb_hive = sc.sticky["honeybee_Hive"]()
     
-    HBSurfaces = hb_hive.callFromHoneybeeHive(HBSurfaces)
+    try:
+        HBSurfaces = hb_hive.callFromHoneybeeHive(HBSurfaces)
     
+    except:
+        
+        w = gh.GH_RuntimeMessageLevel.Warning
+        warn = "One of the surfaces you entered is not a Honeybee surface! \n"+\
+        "please make sure all surfaces entered are Honeybee surfaces! This component will not run until you do!"
+        ghenv.Component.AddRuntimeMessage(w,warn )
+        return
```

Comment:
To keep it consistent I think we should return -1 and  check before assigning the results. I know that this component is not written in that way but it should have.